src/library.rs

```diff
@@ -192,20 +215,25 @@ pub struct Alias {
     pub c_identifier: String,
     pub typ: TypeId,
     pub target_c_type: String,
+    pub doc: Option<String>,
```

Comment:
Appending this to docs with a `# Deprecated` subheading should be workable. Not in the parser though. `#[deprecated]` is better left for some other day. ;)